### PR TITLE
[script.module.requests] 2.27.1+matrix.1

### PR DESCRIPTION
**Pull request by sbroenne**
### Description

Update of [requests](https://pypi.org/project/requests/) to latest version 2.27.1

#### Updated dependencies

Module is dependent on PR #2183 for **charset-normalizer** and PR #2184 for **idna** - add-on check will fail because of this.

From [HISTORY.md](https://github.com/psf/requests/blob/main/HISTORY.md):

#2183 

Instead of [chardet](https://github.com/xbmc/repo-scripts/tree/matrix/script.module.chardet), use the MIT-licensed **charset_normalizer** for Python3 to remove license ambiguity for projects bundling requests. If chardet is already installed on your machine it will be used instead of charset_normalizer to keep backwards compatibility. (#5797).

#2184 

Requests now supports `idna` 3.x on Python 3. `idna` 2.x will continue to be used on Python 2 installations. (#5711)

#### Changes

This module is a simple copy of the upstream module. I have only made updates to the dependencies and the add-on structure (removed duplicate files, changed author to reflect the author of the module).

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### CODE REVIEW

**Comment by L2501** [comment]
v2.27.1 released https://github.com/psf/requests/releases/tag/v2.27.1

File: script.module.requests/addon.xml [hunk 1]
```diff
@@ -1,22 +1,22 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.requests" name="requests" version="2.25.1+matrix.1" provider-name="Python Software Foundation">
+<addon id="script.module.requests" name="requests" version="2.27.0+matrix.1" provider-name="Kenneth Reitz">
```
Comment (L2501) [inline script.module.requests/addon.xml:2]: The requests project was transferred to PSF 2 years ago

Comment (sbroenne) [reply script.module.requests/addon.xml]: I am happy to change that but both the PKG-INFO as well as PyPi lists Kenneth Reitz as the author. I was under the impression that the original author/maintainer was to be used for *provider-name* . Do you want me to change this back to PSF?

Comment (sbroenne) [reply script.module.requests/addon.xml]: Updated version to 2.27.1

Comment (sbroenne) [reply script.module.requests/addon.xml]: @L2501 , how do you want to proceed regarding my last question?

Comment (L2501) [reply script.module.requests/addon.xml]: @enen92

Comment (enen92) [reply script.module.requests/addon.xml]: Keep it as PSF if possible

**Pull request closed without merging**